policyengine_us/parameters/gov/states/ri/tax/income/credits/property_tax/max_amount.yaml

```diff
@@ -0,0 +1,24 @@
+description: Rhode Island provides the following maximum amount under the property tax credit.
+
+values:
+  1977-01-01: 55
+  1978-01-01: 150
+  1979-01-01: 175
+  1980-01-01: 200
+  1997-01-01: 250
+  2006-01-01: 300
+  2007-01-01: 500
+  2022-01-01: 600
+metadata:
+  label: Rhode Island property tax credit maximum amount
+  period: year
+  unit: currency-USD
+  reference:
+  - title: Property Tax Relief R.I. Gen. Laws § 44-33-9
+    href: http://webserver.rilin.state.ri.us/Statutes/TITLE44/44-33/44-33-9.htm
+  - title: RI-1040 Resident Income Tax Return (2022)
```

Comment:
This parameter is not in this tax form. You may need to cite instructions. Please cite the specific page and line number if available to facilitate verification.

Reply:
Hi Max, I think the max property tax amount has been declared in the first reference, 44-33-9, so I just deleted the other reference links. Does that meet your requirements? Or do I need to check for other reference?

Reply:
Please look for a tax form reference as well. It may be in the instruction booklet.